packages/FormElement/src/FormElement.js

```diff
@@ -1,118 +1,47 @@
 import React from "react";
 import PropTypes from "prop-types";
-import * as constants from "@paprika/constants/lib/Constants";
-import extractChildren from "@paprika/helpers/lib/extractChildren";
+import nanoid from "nanoid";
 import isNil from "lodash/isNil";
-import uuidv4 from "uuid/v4";
-
+import * as constants from "@paprika/constants/lib/Constants";
+import * as sc from "./FormElement.styles";
+import { FieldsetContext } from "./components/Content/Content";
+import Content from "./components/Content";
 import Description from "./components/Description";
+import Error from "./components/Error";
 import Instructions from "./components/Instructions";
-import Content from "./components/Content";
-import ErrorMessage from "./components/Error";
-import Help from "./components/Help";
 import Label from "./components/Label";
-import * as sc from "./FormElement.styles";
+import Layout from "./components/Layout";
 
-const subComponentDisplayNames = {
-  Error: "FormElement.Error",
-  Description: "FormElement.Description",
-  Content: "FormElement.Content",
-  Instructions: "FormElement.Instructions",
-  Help: "FormElement.Help",
-};
+export const FormElementContext = React.createContext({});
 
 function FormElement(props) {
-  const {
-    children,
-    hasOptionalLabel,
-    hasRequiredLabel,
-    id,
-    isDisabled,
-    isInline,
-    isLabelVisuallyHidden,
-    label,
-    size,
-    hasFieldSet,
-    onClickLabel,
-    ...moreProps
-  } = props;
-
-  const extractedChildren = extractChildren(children, Object.values(subComponentDisplayNames));
-  const ariaDescriptionId = React.useRef(uuidv4()).current;
-  const ariaErrorId = React.useRef(uuidv4()).current;
-  const ariaInstructionsId = React.useRef(uuidv4()).current;
-  const uniqueInputId = React.useRef(uuidv4()).current;
-  const hasError =
-    !!extractedChildren[subComponentDisplayNames.Error] &&
-    !!extractedChildren[subComponentDisplayNames.Error].props.children;
-
+  const { id, children, isDisabled, size, hasFieldSet, ...moreProps } = props;
+  const [ariaDescribedBy, setAriaDescribedBy] = React.useState({});
+  const uniqueInputId = React.useRef(nanoid()).current;
```

Comment:
This should be `React.useState(nanoid)` is how their library suggested to use it.